core/domain/html_cleaner.py

```diff
@@ -509,6 +509,18 @@ def convert_to_ckeditor(html_data):
         while li.parent.name in ['li', 'p']:
             li.parent.unwrap()
 
+    # Ensure li is wrapped in ol/ul.
+    for li in soup.findAll('li'):
+        if li.parent.name not in ['ol', 'ul']:
```

Comment:
Why not use the LIST_TAGS variable defined below?

Reply:
Done!